src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +44,21 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
-createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#if (!MIN_VERSION_http_conduit(2,1,11))
+  where
+    parseUrlThrow = parseUrl
+#endif
 
 getResponse :: OAuth -> Credential -> Request -> IO (Response BS.ByteString)
-getResponse oauth credential req =
-  withManager $ \m -> do
-    signedreq <- signOAuth oauth credential req
-    httpLbs signedreq m
+getResponse oauth credential req = do
+  m <- newManager tlsManagerSettings
+  signedreq <- signOAuth oauth credential req
+  httpLbs signedreq m
+#if (!MIN_VERSION_http_conduit(2,1,11))
+    where
+      tlsManagerSettings = defaultManagerSettings
```

Comment:
@CristhianMotoche Are `defaultManagerSettings` and `tlsManagerSettings` different? I guess we can simply add http-client-tls as a dependency and import `tlsManagerSettings` from Network.HTTP.Client.TLS (and probably add explicit imports for Network.HTTP.Conduit to hide the re-exported `tlsManagerSettings`). Would that be better?